src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -185,7 +251,7 @@ def reverse_resolve(self, use_case_id: UseCaseKey, org_id: int, id: int) -> Opti
 
         Returns None if the entry cannot be found.
         """
-        table = self._table(use_case_id)
+        table = TABLE_MAPPING[use_case_id]
```

Comment:
I prefer this being wrapped in a method

Reply:
Before `_table` is just
```python
def _table(self, use_case_id: UseCaseKey) -> IndexerTable:
        return TABLE_MAPPING[use_case_id]
```
Do you want me to restore that? Or are there more checks that I should add

Reply:
restoring that works for me

Reply:
fixed